librz/core/cmd_descs/cmd_print.yaml

```diff
@@ -75,15 +75,18 @@ commands:
         entries:
           - text: "pch @! 64 @e:cfg.bigendian=true"
             comment: "Generate a C 32 bits array in big endian format, using 64 bytes"
-          - text: "pcp @! 1024"
+          - text: "pcp 1024"
             comment: "Generate a Python byte array of size 1024"
-          - text: "pcj @! 10"
+          - text: "pcj 10"
             comment: "Generate a JSON bytes array of size 10"
     subcommands:
       - name: pc
         summary: Generate a C/C++ byte array.
         cname: cmd_print_byte_array_c_cpp_bytes
-        args: []
+        args:
+          - name: N
```

Comment:
can we use `len` for this argument? I think this is what it's usually called in many `p?` and `px?` commands.

Reply:
why resolved? i still see `N`.